pkg/client/shub/pull.go

```diff
@@ -23,7 +23,7 @@ const pullTimeout = 7200
 
 // DownloadImage will retrieve an image from the Container Singularityhub,
 // saving it into the specified file
-func DownloadImage(filePath string, shubRef string, force, noHTTPS bool) (err error) {
+func DownloadImage(filePath string, shubRef, tmpDir string, force, noHTTPS, noCache bool) (err error) {
```

Comment:
Why add these parameters if they are unused?

Reply:
Because I was starting on the noCache for shub, but since shub dose not cache right now, I left it, will remove now.